custom/abt/reports/late_pmt.py

```diff
@@ -212,12 +220,15 @@ def not_in_group(key, group):
 
             for date in dates:
                 for user in users:
-                    key = (date.date(), user['user_id'])
-                    if not_in_group(key, group_a) and sub_status != 'group_b':
-                        group = _('No PMT data Submitted')
-                    elif not_in_group(key, group_b) and not not_in_group(key, group_a) and sub_status != 'group_a':
-                        group = _('Incorrect PMT data Submitted')
-                    else:
-                        continue
-                    rows.append(_to_report_format(date, user, group))
+                    if self._user_is_active(user):
```

Comment:
This will perform a DB query for every user for every date.  Can you instead filter out deactivated users in the report data source?
https://github.com/dimagi/commcare-hq/blob/3cda64f2f492ee9be4ff185625aa1394ef5054c8/custom/abt/reports/data_sources/late_pmt.json#L34-L34